chain/client/src/client_actor.rs

```diff
@@ -1117,21 +1117,27 @@ impl ClientActorInner {
             }
         }
 
+        let prev_block_hash = &head.last_block_hash;
         for height in
             latest_known.height + 1..=self.client.doomslug.get_largest_height_crossing_threshold()
         {
             let next_block_producer_account =
                 self.client.epoch_manager.get_block_producer(&epoch_id, height)?;
 
             if me == next_block_producer_account {
```

Comment:
nit as we are touching this anyway - `if me != next_block_producer_account { continue; }` helps to reduce indentation